modules/tensor_mechanics/test/tests/postprocessors/tests

```diff
@@ -16,4 +16,13 @@
     design = 'MaterialTensorAverage.md'
     issues = '#15424'
   [../]
+
+  [./normal_displacement]
+    type = 'Exodiff'
```

Comment:
There should be multiple variants of this test using cli_args that test all the options. There's un-tested code how this is now.